docs/book/intro.md

```diff
@@ -1,3 +1,132 @@
 # zend-mvc-middleware
```

Comment:
For consistency between [navigation](https://github.com/zendframework/zend-mvc-middleware/blob/3daf9e6e05d9c9e615e1dda44016f1579b9d3889/mkdocs.yml#L5) and page, the headline should be "Introduction".